src/core/components/InstallButton/index.js

```diff
@@ -22,29 +24,49 @@ export class InstallButtonBase extends React.Component {
     status: PropTypes.string.isRequired,
   }
 
+  static defaultProps = {
+    clientSupportsAddons: _clientSupportsAddons,
+  }
+
   installTheme = (event) => {
     event.preventDefault();
     const { addon, status, installTheme } = this.props;
     installTheme(event.currentTarget, { ...addon, status });
   }
 
   render() {
-    const { addon, className, hasAddonManager, i18n, size } = this.props;
+    const {
+      addon,
+      clientSupportsAddons,
+      className,
+      hasAddonManager,
+      i18n,
+      size,
+    } = this.props;
     const useButton = hasAddonManager !== undefined && !hasAddonManager;
     let button;
+    const buttonIsDisabled = !clientSupportsAddons();
+    const buttonClass = classNames('InstallButton-button', {
+      'InstallButton-button--disabled': buttonIsDisabled,
```

Comment:
Hmm... I know we use classes everywhere but I feel like `class="button-disabled"` is the butt of many jokes about CSS frameworks. But it's consistent with our other stuff and I like that we keep our targets from a lot of cascading, so I think this is fine.